Tests/test_file_xvthumb.py

```diff
@@ -1,13 +1,35 @@
-from helper import unittest, PillowTestCase
+from helper import hopper, unittest, PillowTestCase
 
-from PIL import XVThumbImagePlugin
+from PIL import Image, XVThumbImagePlugin
+
+TEST_FILE = "Tests/images/hopper.p7"
 
 
 class TestFileXVThumb(PillowTestCase):
 
+    def test_open(self):
+        # Act
+        im = Image.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "XVThumb")
+        self.assert_image_similar(im, hopper("P"), 49)
```

Comment:
49 is not exactly uncorrelated, but it's not a recognizable match either. FWIW, ~85 is totally uncorrelated. 

`assert_image_similar` doesn't actually do palettes.  It converts the data channels to 'L' mode, and then does the diffs on that. The image palettes aren't taken into account. 

`assert_image_equal` does, but only because it runs through `im.tobytes()` rather than `im.getdata()`.

In this case, the correct method is probably to convert both of them to `RGB` first or to quantize `hopper('RGB')` to the funky RGB;332 palette.